src/tools/MappingTools.ts

```diff
@@ -41,32 +42,70 @@ export function mapFile(sourceFile: ts.SourceFile) {
           break;
         case ts.SyntaxKind.InterfaceDeclaration:
           file.addInterface(
-            mapInterface(<ts.InterfaceDeclaration>child, sourceFile),
+            mapInterface(<ts.InterfaceDeclaration>child, sourceFile)
```

Comment:
Reformat changes should not be added to the same commit of the actual code changes.

Reply:
I will remember it for the future, thanks